.github/resources/mariadb/certs-secret.yaml

```diff
@@ -0,0 +1,13 @@
d003SVpxRXF4YlpQRQo2ODI5ajd3OXVyOGhIS2tjSnBGRHVpRFlmY1NwdEE0c1l3cSs5ZTRVZHg5OEJxS1Z5RTMybXRlSkhIL2xPUmVMCkxPRzVqVW40eXpQSW1sa1hlbU9iaEM0ZTFsTWFYZlFwQjN5dzRPTHowemprR2g4b0M4UGNjR0RCNDJoQmdncXUKb1JqUms1eUNXNlFHekZkNStabW1Lb09IQXdQeFpCbU9INlBXTTNDczhqOVhkTUI1L09SOG5ycXFTSGU4RVZ6YwpHNHM2R1lURkpKeUlWOS9ORmxTVml4V3p3L0dCRTNKYXhENnQ1NGI5WmhKYmJ6TzQ0SW1FVGk1cldxeXlOYndJCnpRMGJQVFoyNkRnYWpka3UvbVFOZ2xzTTI1ejdxT3FEME1rUVBPRGZxUnExRDJuU0ZHQlc0NFRnZmV3YWs5VnMKdHZVN1gxMWFva0hKeStJU2lscVR5LzMyazd4MkdnQmhhYjQ3YlU4VkRpNDV5OCtlQWdjQk04SGpHdlJQeXhHMwpnRVJkNjJTTnFHclB6RUJFYmZzcGc0TWVTS2JSZ3NIam5rTXA5Sk10QVU2WjdSaEZTUUtDQVFFQS9RWSs4RVNECkpwZDdCajEvNHNVVXhmYldHTXVlNVJXOUZHUExxNDR4RWJVU056aXFsS0RqRU5vcnpmY1VHTzVVMy84VjB6Q2UKY1JNYkh0WFh6NEExcHNvV0dtSDVDZzRpYzJoMmFJY0ROQ0x6ZkN3NkxHUE45RjVFRDlwQlBYeCtidW1BUVJqTQp1bjBIZEhTVU1UTjljeFNVaEZOTmZQb1JsY0p2STdCa0lkbjZFY3gyNXpzaW1hUmJQV0ZGNHdzeXVKKzFnbVhXCmpCUjg3S01RZnNCLzZiVUQxRjR6VE9qT0FMdnc3Um12bmtuVWhjWjRPZmFrTERCUkVaY0YyQ3ZFYm1TYnRCOVgKOFFXUUY0TWU4ZjJHTTNuT0xYREF6WmFpaTMrdjlWWFVxQmhMcEh1Y0FyWVNQOHZqY21ZcHVRSDZNbmxwbnNsMwpDeDFFUndvRlVDUHNwd0tDQVFFQTFCTGw2dFhvcUh3ODVIcUtWZWlxWnNHQTIxa2ExMEhnT0U2NEtQSlpFc25XCkRoZ1Y3Zk1UTmVzUUNITHA3TEZUeitEVzYyaUtXRmI4WnE3OWJLNlZ0NGE5M3JNb2NjZmxsZFdlOGZMOVRBYlAKaFZsY25nNmhRNEdvTEpyMXRZUG9pRjFycUsxT0hWVGlGUDdlSlYxMlh0ZTN0ejY4ZEVlZ0RRb2ZsLzV5dnE4bwo4RHovUnlETU53MEJ6WGtzc01LYkY1UC9zc2wrdUQyR2RzbG1td1lOUm1TNHhJM3hwcnl3a0I4cTMrUXBrUStkCm1rdER2bjlMQ3FUVHRScW11Z05lUmxxRlRFRFZ4M1VjWlBPWkFoTDAybHlwRzNjcHd3V3dwMDhFekVXYk00Q0sKeVlzSEhUVjlTNC9pVWJqenFtMnRxZVB5a2dFcVNqNU56dGgvM28xQit3S0NBUUFwUExCYlJqODl0ZWgxL3VYdAoxb0xwTks1MkFoR1djbkdMZGl4R2ZIa2cwcXJ5VndqdUNqNlhpNGNpSFVlb1BJZVdyaU1uZFVBTlc3akswTDFJCmN6UExTQnBNR0hXWEh2Q2NjSVVEeWlWS2hQdTczQlZyRUtVTFU2V3lFN2NZUjBidjJweHRmNVkyT3BSUmRZS0YKZGtmV3pwN1o1aDIxNWFlaUNidEZUTVUxQXlxUFpneG02U3RkY1B6eHh2MjE0Q2hYRUtoclpSK1BXdk4xT2FwQgpqaXdLc2RhNjhOTlprSU9xZVBidUhZYVN2S09uWGM1UVhCTXJwSUJCNXFRc05BM2lsQUVRWFI5ZkI3SVJNdXlECk1mZUNRakxKRXh4SkpXaStaWVFPN3ZzbW5qY055cHVGa1pqdHp0UDEzQkttdFZ6S2h5RWdvT01HNjdFcUJzY1EKNVpuM0FvSUJBR1JYRExHdnhBVHp3MXZYSHR6RzJOa0ZkUkpUZG4raWk1UVlxc20wNExBakQwUmI0UVJVbmlKRQp0YkZKcmFSdndsZjQ2YmM0SHp6czZ6OHFUSDIvbk5iWXliMDB4MDRPWm5JZVFMWDVZaDgybWJsZUxwUWFGL0w0CnhibFZHQ3hwM1B1b04rZjNGNTZKS3U4RUo5ZEdBVnVlZFM0ZVlNd2psM2FUODU3V3BUUFRiR3MrY1FvV1dva3gKZmtqWnJHdDFxQ3ZmdW51OUdsZlJ4RzB2blh4N3dJd3QyNzRqNnRJWk5QRDd6aTFncHNvYnVWWG5ob1g3ZEdYMAoxclk1aGVrODdyNGExamJkOWtnSkFRWkN1NEpmdGJvYlhXZkJXZUVNTUNqNUdpd01Rd0o3THk5c3VGRDVuZTdDCmhUSmxiK1NUUURmQndtT1NhWS8zS3BsVGxpeVdhVUVDZ2dFQUpoeHdKeE12dHpQZTliVDU5NWFqVDN1aVpsQ1AKVTJIbTRtQUwrWlFZaG9hcGM5WWhnR2dhbDF0ZlJsNmxBd1pSdlFqSlF1enBoM2FSQlpGbkZaVUxCSjZ5QU9LKwozLy9FYUdrSXphU2tablAvUjFteklrVkExeWRHMG5mNTJONFJjdzEzV054R2phVFA3M0hqV2VydnVhNkt4N29jClR5WG42Tm9KSVpTQW8rT0liaU9lUVc1dlc2L0g5d0VTV1QyYmYwY0tGaGxmZDF0SE5OZHgzejFUVnloZ3FjLzEKNlNMbXpBUzdWeXRMeTBGRTF4azlNek9CMThRZFhWenQ3b051cFZjSzhOMmJZdWQxazh1UzloYTRqZ1BKbnY0SAovdGNiZjNna0h2QTV1VnRLWU9hVWc3WDF4NmhOaFRWN2JvdXRKays4ZndGVjZQS2pPRzF1Si9ic293PT0KLS0tLS1FTkQgUlNBIFBSSVZBVEUgS0VZLS0tLS0K
```

Comment:
assuming this is a mock cert, at a minimum we need to add this file to a .gitleaks.toml or else we will get spammed by security scanners

Reply:
Done, added all configmaps and secrets containing certs and key to a `.gitleaks.toml`.